I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 31, in script
    $ inv.append("roobeer")
  File "game/script.rpy", line 31, in <module>
    $ inv.append("roobeer")
NameError: name 'inv' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 31, in script
    $ inv.append("roobeer")
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 31, in <module>
    $ inv.append("roobeer")
NameError: name 'inv' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
PNW_Business_Alliance 1.0
Tue Mar 29 20:39:55 2022
